### PR TITLE
topology: max98373-rt5682: define dmic 48k core id

### PR DESCRIPTION
**Pull request by fuyuntsuo**
Define DMIC_PIPELINE_48k_CORE to assign dmic 48k pipeline core id and avoid impact to other platforms.

Signed-off-by: fy.tsuo <fy.tsuo@intelli-go.com>

### CODE REVIEW

File: tools/topology/platform/intel/intel-generic-dmic-kwd.m4 [hunk 1]
```diff
@@ -38,6 +38,9 @@ ifdef(`KWD_PIPE_SCH_DEADLINE_US',`',`fatal_error(note: Need to define schedule f
 # define(DMIC_DAI_LINK_48k_NAME, `dmic01')
 ifdef(`DMIC_DAI_LINK_48k_NAME',`',define(DMIC_DAI_LINK_48k_NAME, `dmic01'))
 
+# define(DMIC_PIPELINE_48k_CORE_ID, 0)
+ifdef(`DMIC_PIPELINE_48k_CORE_ID',`',define(DMIC_PIPELINE_48k_CORE_ID, 0))
```
Comment (lgirdwood) [inline tools/topology/platform/intel/intel-generic-dmic-kwd.m4:42]: Where is this being set in the Makefile ?
Btw, can you add `why` we need this change in the commit message. What feature are we enabling.

Comment (fuyuntsuo) [reply tools/topology/platform/intel/intel-generic-dmic-kwd.m4:42]: Sure. Just added more description to the commit message.

DMIC_PIPELINE_48k_CORE_ID is set in `sof-tgl-max98373-rt5682.m4` and this line provides default value (=0) if the upper topology doesn't define it.

**Comment by keyonjie** [comment]
The change looks good to me, Thanks @fuyuntsuo can you add a "Fixes" tag to the commit message as @lgirdwood query about why we need this change?

**Comment by fuyuntsuo** [comment]
> The change looks good to me, Thanks @fuyuntsuo can you add a "Fixes" tag to the commit message as @lgirdwood query about why we need this change?

Done. The website doesn't show git tag in PR rite? Though I see it in my repo.

**Review by keyonjie — APPROVED** [review]

**Comment by lgirdwood** [comment]
@RDharageswari good for you ?

**Review by cujomalainey — APPROVED** [review]

**Pull request merged**